### ISSUE TITLE
middle mouse paste does not work in prompt

### ISSUE THREAD

**Issue by ff2000** [aidanholm/luakit#73]
In the follow entry box it works.

**Comment by aidanholm** [comment]
Unfortunately I can't reproduce this as my setup doesn't have middle click paste at all. Are there any error messages or differences in verbose output?

**Comment by ff2000** [comment]
No, there aren't any error messages, neither with nor without verbose.
It's likely your system is configured to emulate ScrollWheel-click by right-btn+left-btn (press them at te same time). (Or did you mean you simply can't paste with middle click?)

**Comment by aidanholm** [comment]
Simply can't paste. I _can_ paste with Shift+Insert in the follow mode entry box, and not in the command mode entry box, so I can probably track down the bug.

**Comment by aidanholm** [comment]
Actually, my shift-insert issue is caused by the readline keybinding in `config/binds.lua`. Perhaps adding logging to bind activation would help track this down...

**Comment by Ygrex** [comment]
it is likely to be the same issue, here is how it is reproducible for me:

```
$ ./luakit -Up test
$ xsel | wc -c
0
```

with a pointer select URL shown in status bar: «https://luakit.github.io/luakit/»

```
$ xsel | wc -c
32
```

type 'o', «:open » prompt appears

```
$ xsel | wc -c
0
```

**Comment by aidanholm** [comment]
Nice catch; issue was caused by focusing the input bar with `input:focus()` automatically selecting all text. Fix pushed at c48d092
